ui/components/app/confirm-gas-display/confirm-legacy-gas-display/confirm-legacy-gas-display.stories.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import mockState from '../../../../../test/data/mock-state.json';
+import configureStore from '../../../../store/store';
+
+import README from './README.mdx';
+import ConfirmLegacyGasDisplay from './confirm-legacy-gas-display';
+
+const store = configureStore(mockState);
+
+export default {
+  title: 'Components/App/ConfirmLegacyGasDisplay',
+
+  component: ConfirmLegacyGasDisplay,
+  decorators: [(story) => <Provider store={store}>{story()}</Provider>],
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
```

Comment:
enhancement suggestion: 
could use `argTypes` to add controls here 

<img width="954" alt="Screen Shot 2023-03-28 at 4 13 17 PM" src="https://user-images.githubusercontent.com/20778143/228343364-74d49fb7-3b82-445c-a696-803ad0edb0b0.png">

Reply:
@digiwand This component does not have any arguments, so I am wondering if we need to add the `argTypes`.

Reply:
oh, good question. I used controls as a tool to update the UI rather than solely being tied to component args. For instance, adding this could help with testing long text in the UI. 

@georgewrmarshall do you happen to have thoughts on this?